go/sflow/doc/sflow_version_5.txt

```diff
@@ -0,0 +1,2803 @@
+
```

Comment:
This file doesn't really need to be in our repo

Reply:
Done